src/main/resources/lang/polish.lang

```diff
@@ -0,0 +1,190 @@
+	update available: Nowa wersja Skripta dostępna: <gold>%s<reset> (obecnie na tym serwerze jest <gold>%s<reset>)
+	downloading: Pobieranie Skripta <gold>%s<reset>...
+	download in progress: Obecnie trwa pobieranie najnowszej wersji Skripta.
+	download error: <red>Podczas pobierania najnowszej wersji Skripta wystąpił błąd:<light red> %s
+	downloaded: Najnowsza wersja Skripta została pobrana! Zrestartuj serwer lub użyj polecenia /reload aby zastosować zmiany.
+	internal error: Wystąpił wewnętrzny błąd podczas sprawdzania najnowszej wersji Skripta. Sprawdź konsolę lub logi serwera po więcej szczegółów.
+	custom version: Na twoim serwerze jest obecnie zmodyfikowana, nieoficjalna wersja Skripta. Aktualizacje nie będą automatycznie instalowane.
+	nightly build: Na twoim serwerze jest obecnie rozwojowa wersja Skripta. Aktualizacje nie będą automatycznie instalowane.
+
+# -- Commands --
+commands:
+	no permission message: Nie masz odpowiednich uprawnień, aby użyć tego polecenia.
+	cooldown message: Używasz tego polecenia zbyt często, spróbuj ponownie później
+	executable by players: To polecenie może zostać użyte tylko przez graczy
+	executable by console: To polecenie może zostać użyte tylko z konsoli
+	correct usage: Prawidłowe użycie:
+	invalid argument: Niepoprawny argument <gray>'%s<gray>'<reset>. Dozwolone są:
+	too many arguments: To polecenie przyjmuje tylko jeden argument %2$s.
+	internal error: Podczas próby wykonania tego polecenia wystąpił wewnętrzny błąd.
+	no player starts with: Nie ma obecnie gracza, którego nick zaczyna się na '%s'
+	multiple players start with: Na serwerze jest obecnie kilku graczy, których nicki zaczynają się na '%s'
+
+# -- Hooks --
+hooks:
+	hooked: Udało się podpiąć do %s
+	error: Nie udało się podpiąć do %1$s. Może się to zdarzyć w przypadku, gdy Skript nie wspiera zainstalowanej wersji %1$s
+
+# -- Aliases --
+aliases:
+	# Errors and warnings
+	empty string: '' nie jest rodzajem przedmiotu
+	invalid item type: '%s' nie jest rodzajem przedmiotu
+	empty name: Alias musi mieć nazwę
+	brackets error: Nieprawidłowe użycie nawiasów
+	not enough brackets: Sekcja zaczynająca się znakiem %s ('%s') musi być zamknięta
+	too many brackets: Znak %s ('%s') zamyka nieistniejącą sekcję
+	unknown variation: Wariacja %s nie została zdefiniowana wcześniej
+	missing aliases: Następujące minecraftowe ID nie mają żadnych zdefiniowanych aliasów:
+	empty alias: Alias nie ma zdefiniowanych tagów ani minecraftowych ID
+	invalid minecraft id: minecraftowe ID %s nie jest prawidłowe
+	useless variation: Wariacja nie ma zdefiniowanych tagów ani minecraftowych ID, więc jest bezużyteczna
+	invalid tags: Podane tagi nie są zdefiniowane w prawidłowym formacie JSON
+	unexpected section: Sekcje nie są tu dozwolone
+	invalid variation section: Sekcja powinna być sekcją wariacyjną, ale %s nie jest prawidłową nazwą wariantu
+	outside section: Aliasy muszą być umieszczone w sekcjach
+
+	# Other messages
+	loaded x aliases from: Załadowano %s aliasów z %s
```

Comment:
If it is possible to have here 1 or 2 aliases, then I'd include all possibilities for "1 alias", "3 aliasy", "5 aliasów", etc.

Reply:
This message currently doesn't support differentiating between singular and plural in the code :(

But I don't think it will be necessary to add since that number should always be at least 500 or more, the number of aliases in the project is I believe way above that, and unless someone deletes them all, we should be fine :)

Too bad there's no distinction for the weird polish system where 1531 is "aliasów" and 1532 is "aliasy" 😄 

Thanks for the suggestion though!

Reply:
This is what I was talking about when translating it is often **difficult to translate** to correct for both versions or meanings but the important thing is that it was tried